lightninglabs/lightning-app
# Export logs

Closes #1109.

### What This Does

Adds a share button to the top right of the logs on mobile that loads the log full log file and pops up the device's native share dialog to be able to send it somewhere else (Most likely to save it as a file or send it in an email.)

For now, the button just doesn't show on the desktop electron app, but maybe a good follow up would be to have this button open the folder the log file is in so that they could easily upload it.

### Steps to Test

1. Open the app on mobile and go to your logs, confirm you see a share button in the top right
2. Press the share button, confirm you are able to export the logs to your preferred app
3. Open the app on desktop and go to your logs, confirm you do not see a share button in the top right

## Screenshots

![log-export](https://user-images.githubusercontent.com/649992/61483334-98ff3380-a96a-11e9-86d7-f774c9c19f45.png)
